modules/apps/29-fee/module.go

```diff
@@ -91,7 +91,7 @@ func NewAppModule(k keeper.Keeper) AppModule {
 }
 
 // RegisterInvariants implements the AppModule interface
-func (AppModule) RegisterInvariants(ir sdk.InvariantRegistry) {
+func (AppModule) RegisterInvariants(sdk.InvariantRegistry) {
```

Comment:
ditto

```suggestion
func (AppModule) RegisterInvariants(_ sdk.InvariantRegistry) {
```

Reply:
prefer using the im?

I'm thinking about your lookup issue.  Hope to open whole thing for review today.